src/NLog.Raven/RavenTarget.cs

```diff
@@ -122,11 +128,22 @@ protected override void Write(IList<AsyncLogEventInfo> logEvents)
             {
                 using (var bulkInsert = _documentStore.BulkInsert())
                 {
-                    for (int i = 0; i < logEvents.Count; ++i)
+                    foreach (var log in logEvents)
                     {
-                        var logEvent = logEvents[i].LogEvent;
+                        var logEvent = log.LogEvent;
                         var logEntry = CreateLogEntry(logEvent);
-                        bulkInsert.Store(logEntry);
+                        if (ExpirationOffsetInDays > 0)
+                        {
+                            var expiry = DateTime.UtcNow.AddDays(ExpirationOffsetInDays);
+                            bulkInsert.Store(logEntry, new MetadataAsDictionary
+                            {
+                                new KeyValuePair<string, object>("@expires", expiry)
```

Comment:
Why not use `Constants.Documents.Metadata.Expires` instead of `"@expires"` ?

Reply:
Changed to available constant.